lib/install/management/appliance.go

```diff
@@ -605,6 +606,57 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	},
 	)
 
+	// Kubelet
+	if conf.KubernetesServiceURL != "" && conf.KubeletConfigFile != "" {
+		vmName, err := vm2.Name(d.op)
+		if err != nil {
+			d.op.Errorf("Failed to get VM name, error: %s", err)
+			return err
+		}
+		kubeletName := fmt.Sprintf("kubelet-%s", vmName)
+		kubeletStarter := executor.Cmd{
+			Path: "/sbin/kubelet-starter",
+			Args: []string{
+				"/sbin/kubelet-starter",
+			},
+		}
+
+		kubeletStarter.Env = append(kubeletStarter.Env, fmt.Sprintf("%s=%s", "KUBELET_NAME", kubeletName))
+
+		plPort := portLayerPort
+		if conf.Diagnostics.DebugLevel > 2 {
```

Comment:
This change has some obvious documentation impact, but this seems like documentation impact we may lose track of. I'm not sure how we track these things, but this seems worth noting somewhere.

Reply:
Example of where we might want to document this: https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/vch_general_settings.html#debug